launch/tier4_perception_launch/launch/object_recognition/tracking/tracking.launch.xml

```diff
@@ -25,7 +33,15 @@
         <arg name="data_association_matrix_path" value="$(var object_recognition_tracking_radar_object_tracker_data_association_matrix_param_path)"/>
         <arg name="tracker_setting_path" value="$(var object_recognition_tracking_radar_object_tracker_tracking_setting_param_path)"/>
       </include>
+
+      <!--radar long range dynamic object tracking merger-->
+      <include file="$(find-pkg-share tracking_object_merger)/launch/decorative_tracker_merger.launch.xml">
+        <arg name="main_object" value="/perception/object_recognition/tracking/objects"/>
+        <arg name="sub_object" value="$(var radar_tracker_output)"/>
+        <arg name="output" value="/perception/object_recognition/tracking/merged/objects"/>
```

Comment:
[imo]
I want the final tracking output to remain as "/perception/object_recognition/tracking/objects" because if it is changed, we will need to update the CI/CD interface.

Reply:
I agree with you. I fixed this issue in https://github.com/autowarefoundation/autoware.universe/pull/5269/commits/4f8b63c11f55eda35a77693e08707b19f5fd141f.